### PR TITLE
Create GWD

### PR DESCRIPTION
**Pull request by GlennMadden**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by dougaitken** [comment]
Hey @GlennMadden 

For inclusion, there needs to be an entry in the README.md file and then a company profile, it looks like you have created a file named `GWD`

You mention open door employment - are you open to hiring employees directly? That is the core of this list - companies who hire remote employees directly. 

Can you clarify both parts and fix up the submission, please?

File: GWD [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Glenn Website Design
+http://glennwebsitedesign.com
+Company 
+Glenn creates custom websites. Glenn is American managing a remote company internet web design company from Indonesia.
+
+Company size - 0 - 20
+
+
+Remote status
+WE hire mostly remote based freelancers from http://freelancer.com but have open door employment.
```
Comment (GlennMadden) [inline GWD]: We only hire remote employee as we do not operate out of our office. I manage the business but I am on the road most of the time myself for various business purposes so I also manage the business remotely most of the year.

Comment (dougaitken) [reply GWD]: Thanks for confirming.

**Comment by dougaitken** [comment]
HI @GlennMadden - can you create the needed files, please? Rename this addition as `GWD.md` then move it into the `company-profiles` folder.
Then add a new line to the main `README.md` file in alphabetical order and create an entry that way. 

Once you've done that, push your changes to this PR and I'll take a look

**Comment by GlennMadden** [comment]
Thank you 

I apologize I am new to your website.

 

I am not sure I still need nor know how to access it if I do “README.md file and then a company profile”

 

 

I changed the wording on the GWD file to …

We only hire remote employee as we do not operate out of our office. I manage the business but I am on the road most of the time 

myself for various business purposes so I also manage the business remotely most of the year.

 

Glenn Madden

http://glennwebsitedesign.com

File: GWD (hunk 1 above)
Comment (GlennMadden) [reply GWD]: I created a new file in the company-profiles folder glennwebsitedesign.md
Then I add a new line to the main README.md file in alphabetical order.

I did not know how to delete or move the GWD file I created ???

**Pull request closed without merging**